### ISSUE TITLE
QueryNetwork not working in HDDL

### ISSUE THREAD

**Issue by leoll2** [openvinotoolkit/openvino#4036]
With OpenVINO 2021.2, invoking `QueryNetwork` raises a not-implemented exception on HDDL device. On CPU, GPU and MYRIAD it works correctly.

For instance:
```
InferenceEngine::Core ie;
ie.QueryNetwork(net, "HDDL");
```
results in:
```
terminate called after throwing an instance of 'InferenceEngine::details::InferenceEngineException'
  what():  [NOT_IMPLEMENTED]  ngraph::Function is not supported natively
```
It used to work in 2020.4, didn't try with 2021.1.

**Comment by ilya-lavrenov** [comment]
It will be available only in 2021.3 release (not released yet). 

This [change](https://github.com/openvinotoolkit/openvino/pull/3687) allows to share `QueryNetwork` implementation between MYRIAD and HDDL plugins.

**Comment by ilya-lavrenov** [comment]
The issue should be resolved as a part of 2021.3
